requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy==1.19.1
 opencv-python==4.3.0.36
+flask
```

Comment:
@guinnesschen Please include version number, you can generate this dependency list in your shell by running `pip freeze > requirements.txt`.